pkg/database/realm.go

```diff
@@ -711,21 +711,51 @@ func (r *Realm) GetCurrentSigningKey(db *Database) (*SigningKey, error) {
 	return &signingKey, nil
 }
 
+// GetCurrentSMSSigningKey returns the currently active SMS signing key, the one marked
```

Comment:
I know this is following the same naming pattern as `GetCurrentSigningKey`, but per https://golang.org/doc/effective_go.html#Getters, this should be called `CurrentSMSSigningKey()`.